File: Server/mobile_o2o_project_local_server_share_231115/src/main/java/com/ssafy/cafe/model/service/NoteService.java
package com.ssafy.cafe.model.service;

import java.util.List;
import com.ssafy.cafe.model.dto.Note;

public interface NoteService {
  
    void insert(Note note);
    
    Note select(Integer id);
    
    void delete(Integer id);
    
    List<Note> selectAll(String id);
    
    void readNote(Integer id);
    
    String getFCMAddNote(String receiverId);
    
}

File: Andrioid/Admin/app/src/main/java/com/ssafy/smartstore_jetpack/api/NoteService.kt
package com.ssafy.smartstore_jetpack.api

import com.ssafy.smartstore_jetpack.dto.Comment
import com.ssafy.smartstore_jetpack.dto.Note
import com.ssafy.smartstore_jetpack.dto.ReComment
import retrofit2.http.*

interface NoteService {

    @POST("rest/note")
    suspend fun insert(@Body note: Note): Boolean

    @PUT("rest/note/{id}")
    suspend fun readNote(@Path("id") id: Int): Boolean

    @DELETE("rest/note/{id}")
    suspend fun delete(@Path("id") id: Int): Boolean

    @GET("rest/note/{id}")
    suspend fun selectAll(@Path("id") id:String): List<Note>

}